fabric/build.gradle

```diff
@@ -35,6 +36,11 @@ dependencies {
     // Fabric API. This is technically optional, but you probably want it anyway.
     modImplementation "net.fabricmc.fabric-api:fabric-api:${rootProject.findProperty("fabric_api_version")}"
 
+    // Mod Menu
+    modImplementation "com.terraformersmc:modmenu:${rootProject.findProperty("modmenu_version")}", {
+        exclude module: 'fabric-api'
```

Comment:
I am curious: is there a benefit to excluding the fapi module from modmenu? It seems like this isn't strictly needed based on me looking at some other projects

Reply:
I'm not entirely sure, but I would imagine that it cuts down on the amount of things that gradle is fetching. It's a practice that I saw other modders doing, so I figured that I'd follow it. If this is the case, I would assume that there's no harm in it.

I've never tested it, though.